openwpm/deploy_browsers/deploy_firefox.py

```diff
@@ -16,7 +19,12 @@
 logger = logging.getLogger("openwpm")
 
 
-def deploy_firefox(status_queue, browser_params, manager_params, crash_recovery):
+def deploy_firefox(
+    status_queue: Optional[Queue],
```

Comment:
The `status_queue` is required, so we don't need optional here.
Thanks for annotating this!
```suggestion
    status_queue: Queue,
```